libs/community/langchain_google_community/vision.py

```diff
@@ -0,0 +1,43 @@
+from typing import Iterator, Optional
+
+from langchain_core.document_loaders import BaseBlobParser
+from langchain_core.document_loaders.blob_loaders import Blob
+from langchain_core.documents import Document
+
+from langchain_google_community._utils import get_client_info
+
+
+class CloudImageLoader(BaseBlobParser):
```

Comment:
Can you change this to be `CloudVisionLoader` so that it has the product name in the class?

Reply:
for me `CloudImageLoader` sounds more obvious for a user since it loads an image by processing it in the Cloud.

being more easy to grasp for a user is probably more important than having the product name in the class name, wdyt?

Reply:
I understand what you're saying, but it should probably be very clear what Google Cloud product is being used.